docs/pages/guides/icons.md

```diff
@@ -41,8 +41,7 @@ import * as Font from 'expo-font';
 import { createIconSet } from '@expo/vector-icons';
 
 const glyphMap = { 'icon-name': 1234, test: '∆' };
-const expoAssetId = require("assets/fonts/custom-icon-font.ttf");
-const CustomIcon = createIconSet(glyphMap, 'FontName', expoAssetId);
+const CustomIcon = createIconSet(glyphMap, 'FontName', 'custom-icon-font.ttf');
```

Comment:
Yeah this does match `react-native-vector-icons` docs now, have you tested to make sure it works the same as before?

Reply:
That's actually the only way it's worked to use the `create` functions in my project! 🙂